apps/core/src/components/staking/StakingCard.tsx

```diff
@@ -35,21 +27,22 @@ const STATUS_COPY: { [key in StakeState]: string } = {
     [StakeState.InActive]: 'Inactive',
 };
 
-interface StakeCardProps {
+interface StakingCardProps {
     extendedStake: ExtendedDelegatedStake;
     currentEpoch: number;
     inactiveValidator?: boolean;
+    onClick?: () => void;
```

Comment:
```suggestion
    onClick: () => void;
```
From what I see we always provide onClick to StakingCard